account/src/lib.rs

```diff
@@ -0,0 +1,47 @@
+//! Support for signature based authorization of actions on a user account
+//! using public key(s) and signature threshold (minimum number of signatures
+//! needed to authorize an action) stored on-chain.
+
+mod storage;
+mod storage_key;
+mod types;
+
+use borsh::{BorshDeserialize, BorshSerialize};
+pub use namada_core::types::account::AccountPublicKeysMap;
```

Comment:
What's the reason for keeping this module in core as opposed to pulliing it all into here?

Reply:
Good q - there's an unfortunate cyclic dependency. `AccountPublicKeysMap` is also used in the `namada_tx` crate for tx signing and sig verification, but because `namada_account` depends on `namada_storage` and `namada_storage` depends on `namada_tx` (for `TxQueue`), we cannot import `namada_account` in `namada_tx`